hrui/js/src/hrui.js

```diff
@@ -91,6 +91,21 @@
   }
 
   /**
+   * Changes the placeholder text of the quicksearch field
+   */
+  function changeQuickSearchFieldPlaceholder () {
+    $('#crm-qsearch .ui-autocomplete-input').attr('placeholder', 'Quick Search');
```

Comment:
Can we do it from the markup?

Reply:
no, the markup is core markup, we can't change it